Dockerfile

```diff
@@ -21,6 +21,6 @@ COPY ${binary} /nfsplugin
 RUN apt update && apt-mark unhold libcap2
 RUN clean-install ca-certificates mount nfs-common netbase
 # install updated packages to fix CVE issues
-RUN clean-install libgmp10 bsdutils libssl1.1 openssl libc6 libc-bin libsystemd0 libudev1 zlib1g
+RUN clean-install libgmp10 bsdutils libssl1.1 openssl libc6 libc-bin libsystemd0 libudev1 zlib1g gzip liblzma5
```

Comment:
Wouldn't it make more sense to update *all* packages to the latest version, instead of waiting for a CVE that tells you that an update is needed?

Not all bugs get a CVE. If the distro thinks that a package needs a security update, then I trust that more than the CVE database.

Reply:
the cve is reported by 
https://github.com/kubernetes-csi/csi-driver-nfs/blob/700097c7e6127c8758baf0049ab1bb195b35036f/.github/workflows/trivy.yaml#L31-L39

Not sure how to install all packages, there are too many in Ubuntu.
And Ubuntu base image would be upgraded every few months, that may eliminate the necessity to install some packages.

Reply:
new image `k8s.gcr.io/build-image/debian-base:bullseye-v1.2.0` is populated, let me adjust the packages a little

Reply:
point is you think that's safe, while that image would be scanned by tools like trivy, and trivy would reports CVE error, so it's better to fix it though some CVEs would not be invoked by this image. Anyway, it's better to fix it if it's reported, that's not big effort.

Reply:
`debian-base:bullseye-v1.2.0` has more CVE fixes, while it's still missing some fixes, PR updated.

Reply:
> Not sure how to install all packages, there are too many in Ubuntu.

`apt update && apt upgrade`?

> Anyway, it's better to fix it if it's reported, that's not big effort.

It's still causing churn in the Dockerfile. Updating all installed packages is safer and never needs to be touched again.

Then when trivy reports the need to update, just tag a new patch release and rebuild the image.

Reply:
thanks, `apt upgrade` works!